pkg/console/subresource/consoleserver/types.go

```diff
@@ -56,10 +56,11 @@ type MonitoringInfo struct {
 
 // Auth holds configuration for authenticating with OpenShift. The auth method is assumed to be "openshift".
 type Auth struct {
-	ClientID            string `yaml:"clientID,omitempty"`
-	ClientSecretFile    string `yaml:"clientSecretFile,omitempty"`
-	OAuthEndpointCAFile string `yaml:"oauthEndpointCAFile,omitempty"`
-	LogoutRedirect      string `yaml:"logoutRedirect,omitempty"`
+	ClientID                 string `yaml:"clientID,omitempty"`
+	ClientSecretFile         string `yaml:"clientSecretFile,omitempty"`
+	OAuthEndpointCAFile      string `yaml:"oauthEndpointCAFile,omitempty"`
+	LogoutRedirect           string `yaml:"logoutRedirect,omitempty"`
+	InactivityTimeoutSeconds int    `yaml:"inactivityTimeoutSeconds,omitempty"`
```

Comment:
I don't think `omitempty` on these fields will work since they're not pointers.

Reply:
they will, in this case if the `InactivityTimeoutSeconds` will be `0`, it will get omitted

Reply:
> they will, in this case if the `InactivityTimeoutSeconds` will be `0`, it will get omitted

I'm skeptical :) But we can look at it separately since all the existing properties have it also.